website/content/docs/operations/metrics.mdx

```diff
@@ -0,0 +1,66 @@
+---
+layout: docs
+page_title: Boundary Metrics
+description: |-
+  Obtain visibility of various components of a running Boundary
+---
+
+## Boundary Metrics
+
+Boundary provides Prometheus metrics for Controllers and Workers at the ops endpoint on port 9203 through the `/metrics` path. See the [sample code](#sample-code) section for an example listener stanza in a `config.hcl` file.
+
+For more information about Prometheus, see the [Prometheus Overview Guide](https://prometheus.io/docs/introduction/overview). The Prometheus Go library is used to define and export metrics from the Boundary system. The metrics uses the [OpenMetric exposition](https://prometheus.io/docs/instrumenting/exposition_formats/#openmetrics-text-format) format.
+
+## Metrics Collected
+
+The following lists the current metrics used, their descriptions, and full metric names. Metric types collected include latency, traffic, and errors.
```

Comment:
I think we can forgo the summary of the "metric types" and just let the table communicate which metrics are available for collection.
```suggestion
The following lists the current metrics exported, their descriptions, and full names.
```